### ISSUE TITLE
Generator and Discriminator loss NAN after 613 epochs

### ISSUE THREAD

**Issue by AsmarMuqeet** [clovaai/stargan-v2-tensorflow#4]
I am trying to train starGan on Alderley Day/Night Dataset to turn day images to night, but after 613 epoch the loss turns NAN, GAN type is gan-gp and every thing is set to default, can you help to elaborate what could be the cause of this divergence

**Comment by AsmarMuqeet** [comment]
![Screenshot_2020-08-06 TensorBoard](https://user-images.githubusercontent.com/40683799/89492809-e3e52680-d7cb-11ea-8930-6e6c77765edb.png)
![Screenshot_2020-08-06 TensorBoard(1)](https://user-images.githubusercontent.com/40683799/89492811-e6e01700-d7cb-11ea-9bca-3e4ee7bff823.png)

**Comment by Mohammadelc** [comment]
Hi @AsmarMuqeet,

Have you found how to handle loss nan?

**Comment by LiKangyuLKY** [comment]
> I am trying to train starGan on Alderley Day/Night Dataset to turn day images to night, but after 613 epoch the loss turns NAN, GAN type is gan-gp and every thing is set to default, can you help to elaborate what could be the cause of this divergence

Hi, I'm trying to download Alderley Day/Night Dataset for research, but I can’t open the web page https://wiki.qut.edu.au/pages/viewpage.action?pageId=181178395 
Would you please share this dataset with me?
